app/components-react/sidebar/NavTools.tsx

```diff
@@ -91,64 +94,159 @@ export default function SideNav() {
     }
   }
 
+  const handleAuth = () => {
+    if (isLoggedIn) {
+      remote.dialog
+        .showMessageBox({
+          title: $t('Confirm'),
+          message: $t('Are you sure you want to log out?'),
+          buttons: [$t('Yes'), $t('No')],
+        })
```

Comment:
i think ideally we want to use antd modals here if possible

Reply:
![logout-modal-middle](https://user-images.githubusercontent.com/69046953/197859939-1353d80f-6b59-400c-a676-3540462ed079.png)
![logout-modal-low](https://user-images.githubusercontent.com/69046953/197859940-10467549-cdb4-411d-9105-f05138151149.png)
If I use an antd modal and I center it, we run into the issue where we can't display components over the editor. But if positioning it below the editor works for you, we can do that.

Reply:
so because our layout is modular we can't reliable assume that the editor is going to be in a specific position, but what we *can* do is use `updateStyleBlockers` like you did with the animation to ensure that while the modal is active we don't show the display

Reply:
Thanks! A doozy but done!